src/BaGet.Azure/BaGet.Azure.csproj

```diff
@@ -1,19 +1,19 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
-
-    <Description>The libraries to host BaGet on Azure.</Description>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Search" Version="5.0.1" />
-    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="9.4.1" />
-    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="2.1.1" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\BaGet.Core\BaGet.Core.csproj" />
-  </ItemGroup>
-
-</Project>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+
+    <Description>The libraries to host BaGet on Azure.</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Search" Version="5.0.2" />
```

Comment:
Could you explain the need for these dependency changes?

Reply:
the most of dependency changes was made because of project was unable to build at that time + upgrade packages to the latest. I did not control these changes, but thanks, will check and fix

Reply:
>> Microsoft.Azure.Storage.Blob v9.4.1 to WindowsAzure.Storage v9.3.1
seems it was installed a pre-release, I try to use only stable versions, so my opinion is to do not use pre-releases without a strong need, for now the latest stable is 9.3.3 and starting from 9.4 package contains a note:
>> NOTE: As of version 9.4.0, this library has been split into multiple parts and replaced:  See ...

I assume this was a reason of a failed build.

My suggestion is to stay with 9.3 for now

Reply:
What were the build issues you saw?

Reply:
I do not remember it was in September, then I have other work and and back to Nugets :) Version I have with my changes I use in our dev environment, seems works perfect - Thanks a lot for this project. I just want to bring something from myself to it )